src/main/kotlin/no/nav/familie/tilbake/integration/økonomi/OppdragClient.kt

```diff
@@ -143,6 +153,25 @@ class DefaultOppdragClient(@Qualifier("azure") restOperations: RestOperations,
         }
     }
 
+    override fun hentFeilutbetalingerFraSimulering(request: HentFeilutbetalingerFraSimuleringRequest)
+            : FeilutbetalingerFraSimulering {
+        logger.info("Henter feilubetalinger fra simulering for ytelsestype=${request.ytelsestype}, " +
+                    "eksternFagsakId=${request.eksternFagsakId} og eksternId=${request.fagsystemsbehandlingId}")
+        try {
+            return postForEntity<Ressurs<FeilutbetalingerFraSimulering>>(uri = URI.create(hentFeilutbetalingerFraSimuleringUri
+                                                                                                  .toString()),
```

Comment:
Hvorfor konverterer du en uri til string, for så å konvertere den til en uri igjen?